L1Trigger/L1TCalorimeter/python/simDigis_cff.py

```diff
@@ -4,53 +4,49 @@
 
 #
 # Stage-1 Trigger
 #
-if stage1L1Trigger.isChosen() and not stage2L1Trigger.isChosen():
-#
-# -  RCT (Regional Calorimeter Trigger) emulator
-#
-    import L1Trigger.RegionalCaloTrigger.rctDigis_cfi
-    simRctDigis = L1Trigger.RegionalCaloTrigger.rctDigis_cfi.rctDigis.clone()
-    simRctDigis.ecalDigis = cms.VInputTag( cms.InputTag( 'simEcalTriggerPrimitiveDigis' ) )
-    simRctDigis.hcalDigis = cms.VInputTag( cms.InputTag( 'simHcalTriggerPrimitiveDigis' ) )
 #
 # - Stage-1 Layer-2 Calorimeter Trigger Emulator, with required converters (Stage-1 mixes legacy and upgrade) 
 #
-    from L1Trigger.L1TCalorimeter.simRctUpgradeFormatDigis_cfi import *
-    from L1Trigger.L1TCalorimeter.simCaloStage1Digis_cfi import *
-    from L1Trigger.L1TCalorimeter.simCaloStage1FinalDigis_cfi import *
-    from L1Trigger.L1TCalorimeter.simCaloStage1LegacyFormatDigis_cfi import *
-    from L1Trigger.L1TCalorimeter.caloConfigStage1PP_cfi import *
-    SimL1TCalorimeter = cms.Sequence(simRctDigis + simRctUpgradeFormatDigis + simCaloStage1Digis + simCaloStage1FinalDigis + simCaloStage1LegacyFormatDigis)
+from L1Trigger.L1TCalorimeter.simRctUpgradeFormatDigis_cfi import *
+from L1Trigger.L1TCalorimeter.simCaloStage1Digis_cfi import *
+from L1Trigger.L1TCalorimeter.simCaloStage1FinalDigis_cfi import *
+from L1Trigger.L1TCalorimeter.simCaloStage1LegacyFormatDigis_cfi import *
+from L1Trigger.L1TCalorimeter.caloConfigStage1PP_cfi import *
+from Configuration.Eras.Modifier_stage1L1Trigger_cff import stage1L1Trigger
+from Configuration.Eras.Modifier_stage2L1Trigger_cff import stage2L1Trigger
+(stage1L1Trigger & ~stage2L1Trigger).toReplaceWith(SimL1TCalorimeter, cms.Sequence(simRctDigis + simRctUpgradeFormatDigis + simCaloStage1Digis + simCaloStage1FinalDigis + simCaloStage1LegacyFormatDigis))
+
 #
 # Stage-2 Trigger
 #
-if stage2L1Trigger.isChosen():
-    # select one of the following two options:
-    # - layer1 from L1Trigger/L1TCalorimeter package
-    #from L1Trigger.L1TCalorimeter.simCaloStage2Layer1Digis_cfi import simCaloStage2Layer1Digis
-    # - layer1 from L1Trigger/L1TCaloLayer1 package
-    from L1Trigger.L1TCaloLayer1.simCaloStage2Layer1Digis_cfi import simCaloStage2Layer1Digis
-    from L1Trigger.L1TCalorimeter.simCaloStage2Digis_cfi import simCaloStage2Digis
+# select one of the following two options:
+# - layer1 from L1Trigger/L1TCalorimeter package
+#from L1Trigger.L1TCalorimeter.simCaloStage2Layer1Digis_cfi import simCaloStage2Layer1Digis
+# - layer1 from L1Trigger/L1TCaloLayer1 package
+from L1Trigger.L1TCaloLayer1.simCaloStage2Layer1Digis_cfi import simCaloStage2Layer1Digis
+from L1Trigger.L1TCalorimeter.simCaloStage2Digis_cfi import simCaloStage2Digis
+stage2L1Trigger.toReplaceWith(SimL1TCalorimeter, cms.Sequence( simCaloStage2Layer1Digis + simCaloStage2Digis ))
+
+def _modifyStage2L1TriggerCaloParams(process):
     from CondCore.CondDB.CondDB_cfi import CondDB
     CondDB.connect = cms.string("frontier://FrontierProd/CMS_CONDITIONS")
```

Comment:
This line modifies the global `CondCore.CondDB.CondDB_cfi.CondDB` object. It would be better to modify a local clone. I can do that either in this PR or separately.

Reply:
OK.  Either way fine with me.

Reply:
@makortel so I suggest that we move forward with this PR and your suggestion is implemented in a further PR